### DISCUSSION TITLE
part-ii/2021-127/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#79] [Question comments]
Part II 2021 Quantum Information and Computation: Paper 3 , Section I, D 

https://questions.tripos.org/part-ii/2021-127/

**Comment by thenewtonium** [comment]
(This is just the proof of no-signalling theorem extended incomplete mmts on Bob's side. Just replace the Born rule with the version using projectors and use that $(\Pi_a )^\dag \Pi_a = \Pi_a$ and $\sum_a \Pi_a = I$.)
